packages/host/app/resources/auto-attached-card.ts

```diff
@@ -1,40 +1,86 @@
-import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
 
 import { Resource } from 'ember-resources';
 
+import { TrackedSet } from 'tracked-built-ins';
+
+import type { StackItem } from '@cardstack/host/lib/stack-item';
+
 import { type CardDef } from 'https://cardstack.com/base/card-api';
 
 interface Args {
   named: {
-    lastTopMostCard: CardDef | undefined; // top-most card from the card stack
+    stackItems: StackItem[];
```

Comment:
I can see why `stackItems` name may be confusing -- maybe it should be `topMostStackItems`. These are just the top most stack items. So if there are 2 stacks, there shud be an array of 2 elements.